pkg/mdl/named.go

```diff
@@ -13,6 +15,10 @@ type Nameable interface {
 }
 
 func NamedOutput(in interface{}) interface{} {
+	if in == nil || reflect.ValueOf(in).IsNil() {
```

Comment:
IsNil will panic if `in` is not a chan, func, interface, map, pointer, or slice value

Reply:
afaik, namedOutput should always get a pointer as of now